packages/react/src/lib/whatInput.ts

```diff
@@ -234,6 +234,23 @@ if (isBrowser() && 'addEventListener' in window && Array.prototype.indexOf) {
   setUp()
 }
 
+/*
+ * set up for document
+ */
+
+export const setUpWhatInput = (target: Document) => {
+  if (isBrowser() && 'addEventListener' in window && Array.prototype.indexOf) {
+    const whatInputInitialized = 'whatInputInitialized'
+    if (target[whatInputInitialized] === true) {
+      return
+    }
+    target[whatInputInitialized] = true
+
+    addListeners(target)
```

Comment:
It should use `window`, we can use `defaultView`: https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView

```suggestion
    addListeners(target.defaultView)
```